ansible/configs/osp-migration/infra.yml

```diff
@@ -185,13 +185,13 @@
     # Use  server.metadata.hostname  instead
     - name: Run infra-osp-create-inventory Role
       include_role:
-        name: infra-osp-create-inventory
+        name: "{{ANSIBLE_REPO_PATH}}/roles-infra/infra-osp-create-inventory"
```

Comment:
Do we actually need the ANSIBLE_REPO_PATH ? Isn't that in the search path anyway?

Reply:
I copied from here:

./configs/linklight-foundations/post_infra.yml:      name: "{{ANSIBLE_REPO_PATH}}/roles-infra/infra-common-ssh-config-generate"

Maybe you are right and we don't needed it

Reply:
@fridim  ?

Reply:
I modified it now, we can try without the path